README.md

```diff
@@ -1,46 +1,36 @@
-# Getting Started with Create React App
+# Portfolio Web Page - Beata Suścicka i Natalia Mateusiak
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Introduction:
 
-## Available Scripts
+We are Beata & Natalia, and this is our portfolio web! We’ve been friends since hight school. We’ve always loved creating  projects together. A year ago we discover a new passion - designing web apps. That’s how we started our jouney with UX/UI design and coding. We’re glad that you visit us!
```

Comment:
```suggestion
We are Beata & Natalia, and this is our portfolio web! We’ve been friends since high school. We’ve always loved creating  projects together. A year ago we discover a new passion - web app design. That’s how we started our journey with UX/UI design and coding. We’re glad that you visit us!
```